packages/firestore/src/protos/update.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Is it possible to write this as a piece of the `gulpfile.js` at the top level. Removing the dependency on `bash` will make it so we can enable windows dev which I am looking in to.

Reply:
If it's a ton of work, I have no beef, but it feels like something that is doable.

Reply:
Possible?  Probably.  Worth it?  Probably not.  Nobody except us should be updating the checked-in protos.

Reply:
My issue is less specific than the specific use case and more just a "lets make our SDK buildable in multiple envs"

Reply:
Feel free to take a crack at it, but I don't think it's worth our time.  Writing the update.sh at all (rather than just listing the steps in the README.md) was probably a questionable investment.

Reply:
(and to be clear, this isn't at all required to build the SDK...)

Reply:
Also note that on windows bash is straightforwardly available:

https://msdn.microsoft.com/en-us/commandline/wsl/install_guide

Reply:
I guess I haven't used Windows bash, so I tend to stick with JS based tooling. @mikelehen if you could just throw a note in to possibly refactor this to the gulp scripts, I'm fine to go ahead.

Reply:
Also it occurs to me that script should be copypastad into the iOS repo (and Android when that happens). The iOS version was manually assembled via copybara before these protos were public but it would be better to just get this from the public repo just like this.

When Android becomes open source it will need this too. Please don't gulpify this.